.travis-before-script.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e $DRUPAL_TI_DEBUG
+
+# Ensure the right Drupal version is installed.
+# Note: This function is re-entrant.
+drupal_ti_ensure_drupal
+
+# Add needed dependencies.
+cd "$DRUPAL_TI_DRUPAL_DIR"
+
+# These variables come from environments/drupal-*.sh
+mkdir -p "$DRUPAL_TI_MODULES_PATH"
+cd "$DRUPAL_TI_MODULES_PATH"
+
+# Download token 8.x-1.x
+git clone --depth 1 --branch 8.x-1.x http://git.drupal.org/project/token.git
```

Comment:
@Berdir This already uses the 8.x-1.x version of token. So this sohuld be good to go, I am sending it for a re-test though.